learn/components/styling.md

```diff
@@ -3,4 +3,309 @@ title: Styling
 order: 5
 ---
 
-{% stub %}
+Web Components have powerful styling capabilities which make them portable and extensible. Styles declared within the
+Shadow DOM serve as a Web Component’s default styling. It’s similar to writing a user-agent stylesheet for your custom
+element and it works quite similarily as well.
+
+## Shadow Encapsulation: Scoped Styles
+
+Styles declared outside of a shadow tree cannot directly select elements within a shadow tree. Conversely, styles
+declared inside a shadow tree cannot directly select elements outside of a shadow tree. For both of these points, there
+are special pseudo-selectors which allow styles to cross the boundary of the Shadow DOM boundary — and we will cover
+them later on — but first we’ll focus on shadow encapsulation and its effect on styling. Here’s a basic demonstration:
+
+```html
+<style>
+  p {
+    color: deeppink;
+  }
+</style>
+
+<p>This text is deeppink and not teal because it is outside of the shadow root.</p>
+
+<fancy-p>
+  <template shadowroot="open">
+    <style>
+      p {
+        color: teal;
+      }
+    </style>
+    <p>This text is teal and not deeppink because it is inside of the shadow root.</p>
+  </template>
+</fancy-p>
+```
+
+The `<p>` element within the shadow tree is not effected by the styles declared outside of the shadow tree.
+
+## Inheritance
+
+Custom elements abide by the same rules of inheritance as other HTML elements. Properties such as `font-size`,
+`font-family`, and `color` are inherited from a parent element by default. So are [CSS custom properties][]. Top-level
+elements within a shadow tree inherit their inheritable properties from the custom element itself (also known as the
+shadow host). Here’s a demonstration of this at play:
+
+```html
+<style>
+  article {
+    color: deeppink;
+  }
+</style>
+
+<article>
+  <h1>This text is deeppink.</h1>
+  <article-meta>
+    <template shadowroot="open">
+      <style>
+        span {
+          font-style: italic;
+        }
+      </style>
+      <span>By Some Person</span>
+    </template>
+  </article-meta>
+</article>
+```
+
+The `<article-meta>` custom element inherits its `color` from the `<article>` element where it is set to `deeppink`. The
+`<span>` element within the shadow tree inherits its `color` from the `<article-meta>` custom element which means the
+value will be `deeppink`.
+
+## Styling elements outside of a shadow tree
+
+In order to be portable, Web Components can provide default styles for a couple types of elements which exist outside of
+its shadow tree: the custom element itself (also known as the shadow host) and slotted elements.
```

Comment:
I think this paragraph needs some refinement. I feel uneasy about "couple types of elements". Perhaps splitting the two concepts would work better... something like:

```suggestion
In order to be portable, Web Components can provide default styles for the element itself (also known as the shadow host).
They can also style slotted elements with some default styles.
```